### ISSUE TITLE
Brand Identity Guidelines Design Kit shows incorrect "Yellow (-1)" color hex code

### ISSUE THREAD

**Issue by hongaaronc** [ppy/osu-wiki#7439]
Wasn't quite sure where to file this issue. I figured the wiki is the closest thing, but the actual issue is with a PDF asset hosted on Drive, with a link to it in the wiki.

This is referring to the asset linked [here](https://github.com/ppy/osu-wiki/blob/master/wiki/Brand_identity_guidelines/en.md?plain=1#L5).

In Page 9 of the document, the color code for "Yellow (-1)" is incorrect. It looks like an inadvertant mistake from copy+pasting, since it's the same hex code as "Light Blue (-1)".

![BrandIdentityError](https://user-images.githubusercontent.com/5133530/171980551-01be9a1b-0228-488a-aa36-b44ebaf5270f.png)

**Comment by TicClick** [comment]
@arflyte please take a look (the doc belongs to you: https://github.com/ppy/osu-web/issues/4996#issuecomment-534363999)

**Comment by Coppertine** [comment]
It was referred whe  it was originally made with the response being that the swatch itself is not used any more
https://github.com/ppy/osu-web/issues/4996#issuecomment-554184420

**Comment by peppy** [comment]
The whole guidelines needs an update, since most has moved to a new colour scheme and figma resources.

**Comment by arflyte** [comment]
Yes, we no longer use it. Here are the new palettes. https://www.figma.com/file/VIkXMYNPMtQem2RJg9k2iQ/Asset%2FColours?node-id=0%3A1

**Comment by TicClick** [comment]
it seems we can't just insert a link to Figma due to licensing https://github.com/ppy/osu-wiki/pull/8061#discussion_r985267217. wonder if manually exporting assets featured in the design kit from Figma and packing them in a new archive would work
